[2025-05-04T18:45:59.668+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-04T18:45:59.711+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.task_first manual__2025-05-04T18:45:57.168472+00:00 [queued]>
[2025-05-04T18:45:59.727+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.task_first manual__2025-05-04T18:45:57.168472+00:00 [queued]>
[2025-05-04T18:45:59.728+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-05-04T18:45:59.757+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_first> on 2025-05-04 18:45:57.168472+00:00
[2025-05-04T18:45:59.766+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1473) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-04T18:45:59.769+0000] {standard_task_runner.py:63} INFO - Started process 1474 to run task
[2025-05-04T18:45:59.768+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'main', 'task_first', 'manual__2025-05-04T18:45:57.168472+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpfpmbdw_p']
[2025-05-04T18:45:59.771+0000] {standard_task_runner.py:91} INFO - Job 137: Subtask task_first
[2025-05-04T18:45:59.848+0000] {task_command.py:426} INFO - Running <TaskInstance: main.task_first manual__2025-05-04T18:45:57.168472+00:00 [running]> on host 3d59bb7e8591
[2025-05-04T18:46:00.010+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='task_first' AIRFLOW_CTX_EXECUTION_DATE='2025-05-04T18:45:57.168472+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-04T18:45:57.168472+00:00'
[2025-05-04T18:46:00.012+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-04T18:46:17.619+0000] {logging_mixin.py:188} INFO - [Row(count=590707)]
[2025-05-04T18:46:21.191+0000] {logging_mixin.py:188} INFO - +------+
| count|
+------+
|590707|
+------+
[2025-05-04T18:46:21.193+0000] {logging_mixin.py:188} INFO - None
[2025-05-04T18:46:24.662+0000] {logging_mixin.py:188} INFO - 590707
[2025-05-04T18:46:24.663+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-04T18:46:24.668+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-04T18:46:24.702+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=main, task_id=task_first, run_id=manual__2025-05-04T18:45:57.168472+00:00, execution_date=20250504T184557, start_date=20250504T184559, end_date=20250504T184624
[2025-05-04T18:46:24.773+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-04T18:46:24.816+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-04T18:46:24.819+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
